alibi_detect/saving/loading.py

```diff
@@ -68,7 +62,7 @@
 ]
 
 
-def load_detector(filepath: Union[str, os.PathLike], **kwargs) -> Detector:
+def load_detector(filepath: Union[str, os.PathLike], **kwargs) -> Union[BaseDetector, ConfigurableDetector]:
```

Comment:
`BaseDetector` -> `Detector`?

Reply:
I am also confused about this one. Are we not meant to be using the protocol's (i.e. `Detector`) rather than the `BaseDetector` class in the return type now?

Reply:
Yeah, I missed some :face_exhaling: